processing/src/main/java/io/druid/segment/BitmapOffset.java

```diff
@@ -40,6 +40,21 @@
 
   private volatile int val;
 
+  public static IntIterator getReverseBitmapOffsetIterator(ImmutableBitmap bitmapIndex)
+  {
+    ImmutableBitmap roaringBitmap = bitmapIndex;
+    if (!(bitmapIndex instanceof WrappedImmutableRoaringBitmap)) {
+      final BitmapFactory factory = RoaringBitmapSerdeFactory.bitmapFactory;
```

Comment:
I don't think there's a better way for concise.

Also this code is just moved from lower down in the same file, it was added as part of the descending select patch.